boundedint/uint256.go

```diff
@@ -32,17 +32,17 @@ type Uint256 struct {
 var minUint256 = big.NewInt(0)
 var maxUint256 = new(big.Int).Sub(new(big.Int).Exp(big.NewInt(2), big.NewInt(256), nil), big.NewInt(1)) // 2^256 - 1
 
-// New creates a Uint256 struct with a minimum initial underlying value
-func New() *Uint256 {
+// NewUint256 creates a Uint256 struct with a minimum initial underlying value
+func NewUint256() *Uint256 {
 	u := new(Uint256)
 	u.value = *new(big.Int).Set(minUint256)
 	return u
 }
 
-// FromUint64 creates a Uint256 struct based on the given uint64 param
+// Uint64ToUint256 creates a Uint256 struct based on the given uint64 param
 // any uint64 is valid as a Uint256
-func FromUint64(base uint64) *Uint256 {
-	u := New()
+func Uint64ToUint256(base uint64) *Uint256 {
```

Comment:
I would keep the `From` in the function name and put the type in the front. e.g. `Uint256FromUint64`. Or since `uint64` is kind of the default unsigned int type in go anyway we could just call that one `Uint256From`. Same for the `Int256` family.

Reply:
i would rather we settled on a package name before having this discussion

Reply:
function renamed to `Uint256From`.

there is no equivalent function for the `Int256` family _yet_